src/EFCore.Abstractions/PrimaryKeyAttribute.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.EntityFrameworkCore.Utilities;
+
+namespace Microsoft.EntityFrameworkCore;
+
+/// <summary>
+///     Specifies a primary key for the entity type mapped to this CLR type. This attribute can be used for both keys made up of a
+///     single property, and for composite keys made up of multiple properties. `System.ComponentModel.DataAnnotations.KeyAttribute`
+///     can be used instead for single-property keys, in which case the behavior is identical. If both attributes are used, then
+///     this attribute takes precedence.
+/// </summary>
+/// <remarks>
+///     See <see href="https://aka.ms/efcore-docs-modeling">Modeling entity types and relationships</see> for more information and examples.
+/// </remarks>
```

Comment:
I tend to try to keep the summary short

```suggestion
/// <summary>
///     Specifies a primary key for the entity type mapped to this CLR type.
/// </summary>
/// <remarks>
///     <para>
///         This attribute can be used both for keys made up of a single property, and for composite keys made up of multiple properties.
///         <see cref="KeyAttribute" /> can be used instead for single-property keys, in which case the behavior is identical. If both
///         attributes are used, then this attribute takes precedence.
///     </para>
///     <para>
///         See <see href="https://aka.ms/efcore-docs-modeling">Modeling entity types and relationships</see> for more information and
///         examples.
///     </para>
/// </remarks>
```